Makefile

```diff
@@ -68,16 +68,16 @@ clear-caches:  ## | Clear Redis caches.
 
 .PHONY: redis-cache-cli
 redis-cache-cli: .env .docker-build  ## | Open Redis CLI to cache Redis server.
-	docker-compose run redis-cache redis-cli -h redis-cache
+	docker-compose run --rm redis-cache redis-cli -h redis-cache
```

Comment:
The `--rm` reduces accumulation of docker container state which overwhelms my hard drive.